test/node/conversation-direct.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const assert = require('assert');
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+const helper = require('./helper');
+Circuit.logger.setLevel(Circuit.Enums.LogLevel.Error);
+
+let client;
+let user;
+let client2;
+let user2;
+let conversation;
+describe('Direct Conversation', () => {
+
+    before(async () => {
+        client = new Circuit.Client(config.bot1);
+        user = await client.logon();
+        client2 = new Circuit.Client(config.bot2);
+        user2 = await client2.logon();
+    });
+
+    after(async () => {
+        await client.logout();
+        await client2.logout();
+    });
+
+    it('should create a direct conversation', async () => {
+        const res = await client.createDirectConversation(user2.userId);
```

Comment:
I guess we cannot easily test the API with the second parameter 'true'.  I guess we could if the user is an admin in which case he could create a user first. Let's leave that for later. We should have a special test tenant, and not the unify tenant, then we could do that.

Actually we might not even support the inviteUser API yet.